ios/RN/RNCameraUtils.h

```diff
@@ -18,6 +18,6 @@
 + (NSString *)captureSessionPresetForVideoResolution:(RNCameraVideoResolution)resolution;
 + (AVCaptureVideoOrientation)videoOrientationForInterfaceOrientation:(UIInterfaceOrientation)orientation;
 + (CGAffineTransform)videoTransformForOrientation:(UIInterfaceOrientation)orientation;
++ (CGImagePropertyOrientation)imageOrientationForFacialTracking:(UIInterfaceOrientation)orientation :(AVCaptureDevicePosition)camera;
```

Comment:
I would just change this method name to be consistent with the others as well as explicitly calling out the second property so it's clear what's expected by the implementation. Something like:

```
+ (CGImagePropertyOrientation)imageOrientationForInterfaceOrientation:(UIInterfaceOrientation)orientation withDevicePosition:(AVCaptureDevicePosition)camera;
```

If there's a way to make this more concise, go for it, but you get the idea. Typical Objective-C pattern for method names describe what it's returning as well as each parameter being passed.